### ISSUE TITLE
no-paths CLI option missing in v3.0.x

### ISSUE THREAD

**Issue by adamcameron** [Behat/Behat#814]
At some stage between v3 betas and release, the `no-paths` CLI option seems to have gone AWOL. Looking through the source there was a bit of reorganisation during the RC cycle, and it seems to not have survived this process (I could not find the specific commit which caused this).

I'm not sure if this was a purposeful omission as the 2.5.3 CLI docs seem to be the most recent, and this still references it, and not finding the exact commit, I could not see if the comment said "removing this because [reason]".

**Comment by everzet** [comment]
I didn't go away, but formatters settings changed drastically since RC. The way you configure any formatter now is either with `--format-settings`:

```
behat --format-settings='{"paths": false}'
```

or with `BEHAT_PARAMS` environment variable:

```
export BEHAT_PARAMS='{"formatters": {"pretty": {"paths": false}}}'
```

Check https://github.com/Behat/Behat/blob/master/features/format_options.feature for examples

**Comment by adamcameron** [comment]
Cool, thanks for the clarification. But from a software stability standpoint, it did "go away" in the sense that the functionality stopped working after upgrading. Different functionality that does the same thing is... *different* ;-) However I get that this is fine according to semver when bumping full revisions.

Perhaps the issue should actually remain open, as I think there is still possibly a shortfall in the docs, at least as far as transitioning from 2.x to 3.x? I did google *a lot*, and had a superficial (too superficial as it turns out!) look through the source code to find out if it had just changed, but found nothing so assumed it had been removed. Is there some "upgrading from 2.x to 3.x" doc I didn't find? 

Above all else, thanks for the quick heads-up, as it gets me moving forward again. And thanks for putting the effort into Behat! :-)
